CHANGELOG

```diff
@@ -68,6 +68,14 @@ Improved testing
 
 Miscellaneous
 -------------
+- Added support for explicit vectorization for the high-performance templated
+  code, which can now take advantage of specific intrinsics classes on the
+  following architectures:
+    - x86 (SSE/AVX/AVX2/AVX512),
+    - Power8 & Power9 (VSX),
+    - BG/Q (QPX).
```

Comment:
BG/Q may be too old to add now. 😄

Reply:
As it was used for the [paper](https://arxiv.org/abs/2004.06722), it might be still useful.